packages/bundler-rpc/src/types/Bundler.types.ts

```diff
@@ -0,0 +1,21 @@
+export type SendUserOpResponse = {
```

Comment:
These types should return only the object mentioned in data field. code and message are returned from the API call, and it should be handled inside the sdk methods. 

If code is anything other than 2xx then sdk should throw an exception with proper message so dapps can handle it at their end.

Types name should be

UserOpResponse = {
    userOpHash: string,
    wait: <mention function type here that return Promise<TransactionResponse>>
}

UserOpGasFieldsResponse = {
        preVerificationGas: string,
        verificationGasLimit: string,
        callGasLimit: string,
        maxPriorityFeePerGas: string,
        maxFeePerGas: string
}

I removed the gasPrice field. Why there is need to gasPrice field? It's not used in creating UserOp

Reply:
code and message are removed and types name are updated. gasPrice filed is just of type1 transaction as bundler just send gasPrice for non 1155 blockchain so we assign gasPrice value to maxFeePerGas and maxPriorityFeePerGas in userOP

Reply:
See the method in EntryPoint.sol as well

```
function getUserOpGasPrice(MemoryUserOp memory mUserOp) internal view returns (uint256) {
    unchecked {
        uint256 maxFeePerGas = mUserOp.maxFeePerGas;
        uint256 maxPriorityFeePerGas = mUserOp.maxPriorityFeePerGas;
        if (maxFeePerGas == maxPriorityFeePerGas) {
            //legacy mode (for networks that don't support basefee opcode)
            return maxFeePerGas;
        }
        return min(maxFeePerGas, maxPriorityFeePerGas + block.basefee);
    }
```

Reply:
> See the method in EntryPoint.sol as well
> 
> ```
> function getUserOpGasPrice(MemoryUserOp memory mUserOp) internal view returns (uint256) {
>     unchecked {
>         uint256 maxFeePerGas = mUserOp.maxFeePerGas;
>         uint256 maxPriorityFeePerGas = mUserOp.maxPriorityFeePerGas;
>         if (maxFeePerGas == maxPriorityFeePerGas) {
>             //legacy mode (for networks that don't support basefee opcode)
>             return maxFeePerGas;
>         }
>         return min(maxFeePerGas, maxPriorityFeePerGas + block.basefee);
>     }
> ```

So bundler don't have to send gasPrice. Instead it should send same gasPrice value in both maxFeePerGas and maxPriorityFeePerGas